app/assets/stylesheets/typography.scss

```diff
@@ -282,7 +282,7 @@ a.subtle-button {
   border-bottom: solid 1px #cdcbc4;
   text-align: center;
   font-size: 12px;
-  padding: 3px 20px;
+  padding: 3px 17px;
```

Comment:
we now have 5 buttons, this was the solution I found to get everything in the same line. The other solution I found was to decrease the font-size but I thought this one was nicer

![image](https://user-images.githubusercontent.com/1999050/32680506-e3116d54-c662-11e7-86f4-55d628e43d8c.png)

Reply:
I'm definitely not a web designer, but I get uneasy when I see things like "everything in the same line", because my experience has been that when you're at the point of tweaking a few pixels off of the padding, it will almost certainly look different on somebody else's machine (e.g., if they use a slightly zoomed text size).

Should we just bump the "X GUIs shown below" text to a newline in all cases?

Reply:
> Should we just bump the "X GUIs shown below" text to a newline in all cases?

I think that would just be converting the `span` to a `div` like:

```diff
diff --git a/app/views/downloads/guis/index.html.haml b/app/views/downloads/guis/index.html.haml
index e7f930d..1f3d15b 100644
--- a/app/views/downloads/guis/index.html.haml
+++ b/app/views/downloads/guis/index.html.haml
@@ -18,7 +18,7 @@
     =link_to "Linux", '#', {:class => "subtle-button gui-os-filter", 'data-os' => 'linux'}
     =link_to "Android", '#', {:class => "subtle-button gui-os-filter", 'data-os' => 'android'}
     =link_to "iOS", '#', {:class => "subtle-button gui-os-filter", 'data-os' => 'ios'}
-    %span#os-filter-count
+    %div#os-filter-count
       %strong 0
       %span.os Linux
       GUIs are shown below ↓
```